src/loaders/ncbi_downloader/ncbi_downloader.py

```diff
@@ -1,74 +1,76 @@
 """
-PROTOTYPE - Download genome files from NCBI FTP server.
-
-usage: ncbi_downloader.py [-h] --download_file_ext DOWNLOAD_FILE_EXT [DOWNLOAD_FILE_EXT ...] --release_ver
-                          RELEASE_VER [--root_dir ROOT_DIR] [--source SOURCE] [--threads THREADS]
-                          [--overwrite]
+usage: ncbi_downloader.py [-h] --download_file_ext DOWNLOAD_FILE_EXT [DOWNLOAD_FILE_EXT ...] --release_ver {202,207,214,80,83,86,89,95}
+                          [--kbase_collection {GTDB}] [--root_dir ROOT_DIR] [--source {NCBI}] [--threads THREADS] [--overwrite]
                           [--exclude_name_substring EXCLUDE_NAME_SUBSTRING [EXCLUDE_NAME_SUBSTRING ...]]
 
+PROTOTYPE - Download genome files from NCBI FTP server.
+
 options:
   -h, --help            show this help message and exit
 
 required named arguments:
   --download_file_ext DOWNLOAD_FILE_EXT [DOWNLOAD_FILE_EXT ...]
-                        Download only files that match given extensions.
-  --release_ver RELEASE_VER
+                        Download only files that match given extensions
+  --release_ver {202,207,214,80,83,86,89,95}
```

Comment:
I assume this is just from the `-h` output? But `release_ver` choices are not hard coded. Instead it's parsed dynamically from the NCBI website every time this script is called. We should probably add a comment here.

Reply:
Updated!